backend/src/apiserver/server/util.go

```diff
@@ -253,27 +269,23 @@ func ValidatePipelineSpec(resourceManager *resource.ResourceManager, spec *api.P
 // (1) a pipeline version is specified in references as a creator.
 // (2) the above pipeline version does exists in pipeline version store and is
 // in ready status.
-func CheckPipelineVersionReference(resourceManager *resource.ResourceManager, references []*api.ResourceReference) (*string, error) {
```

Comment:
Same here. Just wondering why `CheckPipelineVersionReference` is public. Looks like it is only used in this package.